portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsSubmissions.testcase

```diff
@@ -182,14 +182,56 @@ definition {
 	}
 
 	@description = "Verify that the Submit button's default value is restored if the Submit Button Label is left blank"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test SubmitButtonRestoredToDefaultIfLeftBlank {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-131032 SubmitButtonRestoredToDefaultIfLeftBlank pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.addTextField(fieldPositionNumber = "0");
+
+		FormsAdminNavigator.gotoSettings();
+
+		Form.configureSubmitLabel(submitLabel = "Send Form");
+
+		Form.save();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
+
+		Form.viewSubmitButtonText(key_buttonText = "Send Form");
+
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content & Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoEditFormByName(formName = "Form 1");
+
+		FormsAdminNavigator.gotoSettings();
+
+		Form.configureSubmitLabel(submitLabel = " ");
+
+		Button.clickDone();
+
+		Form.save();
+
+		FormsAdminNavigator.gotoPublishedForm();
+
+		Form.viewSubmitButtonText(key_buttonText = "Submit");
+
+		FormPortlet.submitLocalizedSuccessfully(buttonName = "Submit");
```

Comment:
Hey @lucasperj, the last step planned for this test is `Assert the Success Page`, but this line is just asserting that a success message is being displayed. In light of this, can you please assert that the Success Page is also being displayed?

The macro `viewSuccessPage` will do it for you. :slightly_smiling_face:

Reply:
Done!